api/tests/app/letters/test_anyways.py

```diff
@@ -0,0 +1,4 @@
+import pytest
```

Comment:
What's your thinking with the suffix `_anyways`? Is there a better name for it - not clear what your intention is. Is it like a `test_keep.py` or a `test_skip.py`?